parsers/test/test_NORDPOOL.py

```diff
@@ -39,24 +39,12 @@ def test_price_parser_se(adapter, session, snapshot):
         json=loads(mock_data_next_day.read_text()),
     )
 
-    price = NORDPOOL.fetch_price(
+    assert snapshot == NORDPOOL.fetch_price(
         zone_key=ZoneKey("SE-SE4"),
         session=session,
         target_datetime=datetime.fromisoformat("2024-07-08"),
     )
 
-    assert snapshot == [
-        {
-            "datetime": element["datetime"].isoformat(),
-            "price": element["price"],
-            "currency": element["currency"],
-            "source": element["source"],
-            "zoneKey": element["zoneKey"],
-            "sourceType": element["sourceType"].value,
-        }
-        for element in price
-    ]
-
```

Comment:
This PR is mostly changes like these, but included a few other detail changes in files already using snapshots where I transitioned to file-consistent use of snapshots when it made sense.